packages/crud/src/vaadin-crud.js

```diff
@@ -1180,11 +1203,8 @@ class Crud extends ControllerMixin(ElementMixin(ThemableMixin(PolymerElement)))
   }
 
   /** @private */
-  __new(event) {
-    // This allows listening to parent element and fire only when clicking on default or custom new-button.
-    if (event.composedPath().some((e) => e.nodeType === 1 && e.hasAttribute('new-button'))) {
-      this.__confirmBeforeChangingEditedItem(null, true);
-    }
+  __new() {
```

Comment:
**question:** Who calls this method now?

Reply:
This is added as a click listener here, same as for other slotted buttons:

https://github.com/vaadin/web-components/blob/879a5f2368e373ed64124a699888fdf7d24d46b4/packages/crud/src/vaadin-crud.js#L1008-L1016

Reply:
Got it, thanks!